docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -32,8 +37,8 @@ Option                                 Usage
 Output
 ^^^^^^
 
-The output per bin is given in *charge density*.
-To get a *charge of particles per dr_i and dp_i bin* multiply by:
+Values per phase space bin are given over *charge density*.
```

Comment:
it's not really *in* or *as* but *also divided by* (since it's divided by more constants explained in the following sentences)